src/_guides/language/sound-problems.md

```diff
@@ -607,8 +542,8 @@ Do we have any known issues or bugs to list here?
 ### The covariant keyword
 
 Some (rarely used) coding patterns rely on tightening a type
-by overriding a parameter's type with a subtype, which is illegal in strong
-mode Dart. In this case, you can use the `covariant` keyword to
+by overriding a parameter's type with a subtype, which is illegal.
```

Comment:
I prefer "invalid" to "illegal". Just personal peeve, but I don't consider us qualified to pass actual *laws*. We just get to define what is validly Dart.